content/english/python-discord-translate/Setup/code-prerequisites/_index.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "Code Prerequisites"
+draft: false
+weight: 2
+---
+
+Nuvi can’t invite a translator bot yet—there’s no code environment set up! Let’s get your computer ready. You’ll install Python, set up a virtual environment (a safe “bubble” for your project’s packages), install VS Code (your code editor), and create the starter files. Once this foundation is ready, you’ll start writing the bot that helps everyone understand each other.
+
+{{< alert theme="info" >}}
+<img src="../../media/NF_mascot.jpg" alt="Nuvi mascot" width="90" style="float:right;margin:0 0 8px 12px;" />
+<strong>Nuvi says:</strong> Pick Python 3.10 so the translator parts install smoothly—future upgrades are easier once things work now!
```

Comment:
<img width="1149" height="85" alt="Screenshot 2025-09-17 at 4 03 00 PM" src="https://github.com/user-attachments/assets/6500cc54-27f7-42c8-8f49-c3dd0574e504" />

Seems weird - you might need to get the alert shortcode changed